src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,44 @@
+package study;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+
+	private Set<Integer> numbers;
+
+	@BeforeEach
+	void setUp() {
+		numbers = new HashSet<>();
+		numbers.add(1);
+		numbers.add(1);
+		numbers.add(2);
+		numbers.add(3);
+	}
+	
+	@Test
+	public void set_size_확인_success() throws Exception {
+	    assertThat(numbers.size()).isEqualTo(3);
+	}
+
+	@ParameterizedTest
+	@ValueSource(ints = {1, 2, 3})
+	public void parameterized_test_success(int number) {
+		assertThat(numbers.contains(number)).isTrue();
+	}
+
+	@ParameterizedTest
+	@CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+	public void parameterized_with_expected_value_success_or_fail(int value, String expected) {
```

Comment:
```suggestion
	public void parameterized_with_expected_value_success_or_fail(int value, boolean expected) {
```

별도로 파싱하지 않고 이렇게도 사용이 가능합니다!
참고하시면 좋을 것 같아요. 👍